src/components/Todolist.tsx

```diff
@@ -0,0 +1,37 @@
+import { Todo } from '../types/Todo';
+import { TodoInfo } from './TodoInfo';
+
+type Props = {
+  todos: Todo[],
+  deletingTodosIds: number[],
+  temporaryTodo: Todo | null,
+  handleDeleteTodo: (id: number) => void,
+};
+
+export const Todolist: React.FC<Props> = ({
```

Comment:
React.memo
```suggestion
export const TodoList: React.FC<Props> = ({
```

Reply:
done, I made these corrections already in the third part